web/widgets.py

```
@@ -1,3 +1,7 @@
+import html
+import json
+from typing import Any, Dict
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)